apis/management.cattle.io/v3/multi_cluster_app.go

```diff
@@ -39,3 +40,8 @@ type Answer struct {
 	ClusterName string            `json:"clusterName,omitempty" norman:"type=reference[cluster]"`
 	Values      map[string]string `json:"values,omitempty" norman:"required"`
 }
+
+type Member struct {
+	UserName   string `json:"userName,omitempty" norman:"type=reference[user],required"`
```

Comment:
@cjellick suggested that if we are going to support groups, then we have to go with the approach similar to PRTB where we have userPrincipalID and groupPrincipalID